### ISSUE TITLE
AutomationProperties page is missing Name Property for Font Size of text box.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/Xaml-Controls-Gallery#323]
**Describe the bug**

**Steps to reproduce the bug**

1. Launch XAML Controls Gallery.
2. Activate Automation Properties from home page.
3. Navigate to text box present beside Font Size of texts and observe its name property.

**Expected behavior**

Name Property should be there for text box

**Actual behavior**

Text box name property missing.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/72557372-7389ae80-3855-11ea-9acb-7533fcb29ea5.png)

**Version Info**

Gallery version (found on Settings page): 

1.2.12.0

**Additional context**

Copied from [internal bug](https://dev.azure.com/microsoft/OS/_workitems/edit/24649841)

**Comment by chingucoding** [comment]
I think this is an issue with NumberBox. Setting AutomationProperties.Name explicitly, the `edit` still has no name set. It seems like the UIA name is only applied to the "container" of the `edit`.

**Comment by YuliKl** [comment]
@chingucoding you're right. I remember having this discussion on the original NumberBox page PR. Moving this issue to WinUI.
